deploy/acm-policies/50-GENERATED-rosa-ingress-certificate-policies.Policy.yaml

```diff
@@ -43,6 +43,14 @@ spec:
                               scope: '{{hub- if eq (fromConfigMap "openshift-acm-policies" .ManagedClusterName "endpoint-publishing-strategy") "internal" -hub}} Internal {{hub- else -hub}} External {{hub- end -hub}}'
                           {{hub- end hub}}
                     {{- end }}
+                    {{- if ne (lookup "operator.openshift.io/v1" "IngressController" "openshift-ingress-operator" "default").metadata.name "default" }}
+                    - complianceType: musthave
+                      metadataComplianceType: musthave
+                      objectDefinition:
+                        kind: Namespace
+                        metadata:
+                          name: donotcreateme?
```

Comment:
'?' is not a valid character for the name

Reply:
That was the point. We want something that is guaranteed to fail to be applied, which marks the Policy as non-compliant, then when the default IngressController resource is created, this section is NO longer applied be of the IF on line 46.

This seemed safer, then trying to engage a webHook that would for example try to block creating a new ClusterVersion kind resource.

Reply:
If your okay with that, I can ask it to be reviewed and pushed to integration to test.